index.html

```diff
@@ -44,5 +51,137 @@ <h2>I’am a Software Developer</h2>
         <a href="#"><img src="src/Social-Media.svg" alt="social-media" /> </a>
       </div>
     </section>
+
+    <section class="my-works">
+      <div class="works-section-a">
+        <h2>My Recent Works</h2>
+
+        <div class="wrapper">
+          <div class="Feature">
+            <img src="src/Img-Placeholder-n1.svg" alt="Feature stories" />
+            <h3>Multi-Post Stories</h3>
+            <p class="Feat-text">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+            <ul class="tech-stack-1">
+              <li>CSS</li>
+              <li>HTML</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <p class="btn">See Project</p>
+          </div>
+        </div>
+      </div>
+      <div class="proj-container">
+        <div class="projects">
+          <h3>Profesional Art Printing Data</h3>
+          <p class="Feat-text">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul class="tech-stack-2">
+            <li>CSS</li>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <p class="btn">See Project</p>
+        </div>
```

Comment:
- [ ] Hi, I can see you are putting in a lot of effort to fix this section. However, we will still need it to perfectly match the Figma Design.  It's important to keep a keen eye. in order to much the design really well. play close attention to the details and it will make you better 👍. You are almost there just make these small changes to your design.
## Your Design
![image](https://user-images.githubusercontent.com/18730106/143719779-9430ed11-fc94-460d-a489-45a4355f1c93.png)
## Figma Design
![image](https://user-images.githubusercontent.com/18730106/143720104-0a40bd5d-1697-4cdf-adaa-bf6cafc57af2.png)


- [ ] Look at the letter spacing in the description. the one you have is too much
- [ ] Look at the border around the button. you should remove it.
- [ ] there is space around the individual technologies. please create that margin around them.
- [ ] the button shouldn't be sticking out from the bottom 👍.